<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <underline>false</underline>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <widget class="QPlainTextEdit" name="mobileNumber_pt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pass_pt">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>340</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>250</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Mobile Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>320</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>410</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Dont have an account ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="createAccount_lb">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>460</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
     <underline>true</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Create account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>380</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>8</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Forget your password ?</string>
   </property>
  </widget>
  <widget class="QLabel" name="clickHere_lb">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>380</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>8</pointsize>
     <bold>false</bold>
     <underline>true</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Click here </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>50</y>
     <width>301</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>60</pointsize>
     <bold>true</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Chat x </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>190</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cairo</family>
     <pointsize>20</pointsize>
     <bold>false</bold>
     <underline>false</underline>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="text">
    <string>Login </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
